app/models/event.rb

```diff
@@ -39,6 +41,8 @@ class Event < ApplicationRecord
   UserActivated = Struct.new(:partner_id, :team_id, :user_id, :target_user_id, keyword_init: true)
   UserDeactivated = Struct.new(:partner_id, :team_id, :user_id, :target_user_id, keyword_init: true)
   ActiveUserCount = Struct.new(:partner_id, :team_id, :user_id, :active_user_count, keyword_init: true)
+  PaymentPlanCreated = Struct.new(:partner_id, :user_id, :payment_plan_id, keyword_init: true)
```

Comment:
This can be a single event, PaymentPlanModified, the event need more data points which are mentioned in the ticket. Without the datapoints the event is not going to be useful.

Reply:
Data points
    Learning Partner ID
    Plan Start Date
    Plan End Date
    Per-Seat Amount
    Number of Seats